BraveWallet/Crypto/Stores/CryptoStore.swift

```diff
@@ -281,6 +281,9 @@ public class CryptoStore: ObservableObject {
       return true
     }
     let pendingRequest = await fetchPendingWebpageRequest()
+    if self.pendingRequest == nil {
+      self.pendingRequest = pendingRequest
+    }
```

Comment:
should we check the equality rather than checking if its nil?

Reply:
yeah ok. given we hard-coded the order of checking pending request, i doubt it will somehow have a different request.